lib/active_model_serializers/adapter/json_api/meta.rb

```diff
@@ -18,7 +18,7 @@ def initialize(serializer)
 
           # Use the return value of the block unless it is nil.
           if serializer._meta.respond_to?(:call)
-            @value = instance_eval(&serializer._meta)
```

Comment:
Yeah, a different PR would probably be better, as *this* pr solves the immediate issue, and what I'm suggesting is more structural.